src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -533,7 +533,7 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
         cursor,
         limit,
     }: IFeatureSearchParams): Promise<IFeatureOverview[]> {
-        let query = this.db('features');
+        let query = this.db('features').limit(limit);
```

Comment:
this may fix our problem with limit not reflecting number of features but a number of feature env rows